stl/inc/format

```diff
@@ -1579,39 +1649,64 @@ _OutputIt _Write(_OutputIt _Out, const _Float _Value, const _Basic_format_specs<
 
     auto _Append_decimal   = false;
     auto _Exponent_start   = _Result.ptr;
+    auto _Radix_point      = _Result.ptr;
+    auto _Integral_end     = _Result.ptr;
     auto _Zeroes_to_append = 0;
+    auto _Separators       = 0;
+    std::string _Groups;

```

Comment:
Ditto "We don't need to `std::`-qualify the names of types when we're in `std`."